tests/report/common/tests.cpp

```diff
@@ -298,6 +298,7 @@ static oe_result_t oe_get_quote_validity_with_collaterals(
             header->report,
             header->report_size,
             &sgx_endorsements,
+            NULL,
```

Comment:
```suggestion
            nullptr,
```
nullptr since this is a C++ file

Reply:
Thanks for that!